sdk/core/core-client-rest/src/apiVersionPolicy.ts

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { PipelinePolicy } from "@azure/core-rest-pipeline";
+import { ClientOptions } from "./common";
+import { URL } from "./url";
+
+export const apiVersionPolicyName = "ApiVersionPolicy";
+
+/**
+ * Creates a policy that sets the apiVersion as a query parameter on every request
+ * @param options - Client options
+ * @returns Pipeline policy that sets the apiVersion as a query parameter on every request
+ */
+export function getApiVersionPolicy(options: ClientOptions): PipelinePolicy {
+  return {
+    name: apiVersionPolicyName,
+    sendRequest: (req, next) => {
+      if (options.apiVersion !== undefined) {
```

Comment:
presumably other falsy values like `""` are also worth filtering out

```suggestion
      if (options.apiVersion) {
```